schemas/paconn-apiProperties.schema.json

```diff
@@ -0,0 +1,743 @@
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "x-ms-apimTemplate-operationName": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "x-ms-apimTemplate-policySection": {
+              "type": "string",
+              "enum": [
+                "Request",
+                "Response"
+              ]
+            }
+          },
+          "patternProperties": {
+            "^x-ms-apimTemplateParameter.\\w+$": {
+              "description": "By convention, most template parameters are named like this."
+            }
+          }
+        }
+      }
+    },
+    "PolicyTemplateInstance-convertarraytoobject": {
+      "type": "object",
+      "required": [
+        "templateId"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "templateId": {
+          "type": "string",
+          "enum": [
+            "convertarraytoobject",
+            "ConvertArrayToObject"
+          ]
+        },
+        "title": {
+          "$ref": "#/definitions/PolicyTemplateInstanceBase/properties/title"
+        },
+        "parameters": {
+          "type": "object",
+          "required": [
+            "x-ms-apimTemplateParameter.propertyParentPath",
+            "x-ms-apimTemplateParameter.keyWithinCollectionPath",
+            "x-ms-apimTemplateParameter.newPropertyPath"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "x-ms-apimTemplate-operationName": {
+              "$ref": "#/definitions/PolicyTemplateInstanceBase/properties/parameters/properties/x-ms-apimTemplate-operationName"
+            },
+            "x-ms-apimTemplate-policySection": {
+              "$ref": "#/definitions/PolicyTemplateInstanceBase/properties/parameters/properties/x-ms-apimTemplate-policySection"
+            },
+            "x-ms-apimTemplateParameter.propertyParentPath": {
+              "type": "string"
+            },
+            "x-ms-apimTemplateParameter.propertySubPath": {
+              "type": "string",
+              "default": "null"
+            },
+            "x-ms-apimTemplateParameter.keyWithinCollectionPath": {
+              "type": "string"
+            },
+            "x-ms-apimTemplateParameter.newPropertyPath": {
+              "type": "string"
+            },
+            "x-ms-apimTemplateParameter.retainKey": {
+              "type": "string",
+              "enum": [
+                "true",
+                "false"
+              ],
+              "default": "true"
+            }
+          }
+        }
+      }
+    },
+    "PolicyTemplateInstance-convertobjecttoarray": {
+      "type": "object",
+      "required": [
+        "templateId"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "templateId": {
+          "type": "string",
+          "enum": [
+            "convertobjecttoarray",
```

Comment:
Is the difference between the two is just in casing?

Reply:
Added [Issue 196](https://github.com/microsoft/PowerPlatformConnectors/issues/196) to track this.